Source/common/SNTConfigurator.m

```diff
@@ -715,6 +717,20 @@ - (NSString *)syncClientAuthCertificateIssuer {
   return self.configState[kClientAuthCertificateIssuerKey];
 }
 
+- (SNTSyncCompressionEncoding)syncClientContentEncoding {
+  NSString *contentEncoding = [self.configState[kClientContentEncoding] lowercaseString];
+  if ([contentEncoding isEqualToString:@"zlib"]) {
+    return SNTSyncCompressionEncodingZlib;
+  } else if ([contentEncoding isEqualToString:@"gzip"]) {
+    return SNTSyncCompressionEncodingGzip;
+  } else if ([contentEncoding isEqualToString:@"none"]) {
+    return SNTSyncCompressionEncodingNone;
+  } else {
+    // Ensure we have the same
```

Comment:
the same...?

Reply:
That should have read `Ensure we have the same default zlib behavior Santa's always had by default.`

Reply:
I'll fix the wording.